shared/src/business/useCases/workitems/getDocumentQCInProgressForSectionInteractor.test.js

```diff
@@ -0,0 +1,116 @@
+const {
+  applicationContext,
+} = require('../../test/createTestApplicationContext');
+const {
+  DOCKET_NUMBER_SUFFIXES,
+  DOCKET_SECTION,
+  PETITIONS_SECTION,
+  ROLES,
+} = require('../../entities/EntityConstants');
+const {
+  getDocumentQCInProgressForSectionInteractor,
+} = require('./getDocumentQCInProgressForSectionInteractor');
+
+describe('getDocumentQCInProgressForSectionInteractor', () => {
+  let mockWorkItem = {
+    createdAt: '',
+    docketEntry: {
+      sentBy: 'petitioner',
+    },
+    docketNumber: '101-18',
+    docketNumberSuffix: DOCKET_NUMBER_SUFFIXES.SMALL,
+    messages: [],
+    section: DOCKET_SECTION,
+    sentBy: 'docketclerk',
+  };
+
+  it('throws an error if the user does not have access to the work item', async () => {
+    applicationContext.getCurrentUser.mockReturnValue({
+      role: ROLES.petitioner,
+      userId: 'petitioner',
+    });
+    applicationContext
+      .getPersistenceGateway()
+      .getDocumentQCServedForSection.mockResolvedValue(mockWorkItem);
+
+    let error;
+    try {
+      await getDocumentQCInProgressForSectionInteractor(applicationContext, {
```

Comment:
Nit: this can be shorted to `await expect(getDocumentQCInProgressForSectionInteractor...).rejects.toThrow()`